### DISCUSSION TITLE
IQ Testi

### DISCUSSION THREAD

**Discussion by akgembi12** [sonsuzus/sonsuzus#461] [Soru Cevap] [answered]
https://codeforces.com/problemset/problem/25/A

Bob IQ testini geçmeye hazırlanıyor. Bu testteki en sık karşılaşılan görev, verilen n adet sayıdan hangisinin diğerlerinden farklı olduğunu bulmaktır . Bob, bir sayının genellikle diğerlerinden çift-tek olma açısından farklı olduğunu gözlemledi. Bob'a yardım edin — cevaplarını kontrol etmek için, verilen n sayıdan çift-tek olma açısından farklı olanı bulan bir programa ihtiyacı var.

**Giriş**
İlk satır tam sayı  ( 3 ≤  n  ≤ 100 ) içerir ve bu, sorudaki sayı miktarıdır. İkinci satır 100'ü aşmayan n adet boşlukla ayrılmış doğal sayı içerir. Bu sayılardan birinin diğerlerinden çift-tek olma açısından farklı olduğu garantidir.

**Çıktı**
Çift-tek olma açısından diğerlerinden farklı olan sayının indeksi. Sayılar giriş sırasına göre 1'den başlayarak numaralandırılır.

**Örnek**

```
input
5
2 4 7 8 10
```
```
output
3
```
---

```
input
4
1 2 1 1
```
```
output
2
```

**Comment by akgembi12** [comment]
Ali Ekrem KAYLI

**Comment by sonsuzus** [comment]
Çözümü ayrı bir yorum olarak eklersen daha güzel. Soru yukarıda kalsın çözümü aşağıda yorum olarak olsun.

**Comment by akgembi12** [comment, marked as answer]
```
a=input()
b=input()

l=b.split(" ")
liste=[]

for i in l:
    liste.append(int(i))

def ciftmi(x):
    if x%2==0:
        return 1
    else:
        return 0

yenils=[]

for i in liste:
    if ciftmi(i):
        yenils.append(2)
    else:
        yenils.append(1)


if yenils.count(2)==1:
    print(yenils.index(2)+1)
else:
    print(yenils.index(1)+1)

```
